transaction-status/src/lib.rs

```diff
@@ -320,13 +322,16 @@ pub struct UiTransactionStatusMeta {
     pub fee: u64,
     pub pre_balances: Vec<u64>,
     pub post_balances: Vec<u64>,
-    pub inner_instructions: Option<Vec<UiInnerInstructions>>,
+    pub inner_instructions: OptionSerializer<Vec<UiInnerInstructions>>,
     pub log_messages: Option<Vec<String>>,
     pub pre_token_balances: Option<Vec<UiTransactionTokenBalance>>,
     pub post_token_balances: Option<Vec<UiTransactionTokenBalance>>,
     pub rewards: Option<Rewards>,
-    #[serde(default, skip_serializing_if = "Option::is_none")]
-    pub loaded_addresses: Option<UiLoadedAddresses>,
+    #[serde(
+        default = "OptionSerializer::skip",
+        skip_serializing_if = "OptionSerializer::should_skip"
+    )]
```

Comment:
`OptionSerializer` kinda breaks if these attributes are forgotten on a field. How about encapsulating the serializer in a new struct like this: https://github.com/CriesofCarrots/solana/pull/358

Reply:
The encapsulation approach I suggested doesn't actually improve upon anything because it requires adding `#[serde(flatten)]` to attributes. I prefer your original approach or one of the variants suggested in this SO post: https://stackoverflow.com/questions/44331037/how-can-i-distinguish-between-a-deserialized-field-that-is-missing-and-one-that

Reply:
The original approach was almost identical to the last variety (https://stackoverflow.com/a/66434149), with the exception of `#[serde(default = "OptionSerializer::skip")]` to preserve the current serde (json) default behavior. So I reverted to that.